### DISCUSSION TITLE
Adding Syntax Highlighting in Code Blocks

### DISCUSSION THREAD

**Discussion by divineopera** [TriliumNext/Trilium#3342] [Ideas]
Is it possible to integrate Syntax Highlighting in Code Blocks similarly to that in Code Notes?

**Comment by fooyoo** [comment]
see https://github.com/zadam/trilium/discussions/2822

<img src="https://user-images.githubusercontent.com/36770100/203458978-ef986458-3662-49a1-831f-16f0111b9278.png" width="600">
